### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by hazendaz**
----

Make sure these boxes are checked before submitting your PR -- thank you!

- [ ] Added an entry into `CHANGELOG.md` if you have changed SpotBugs code

### CODE REVIEW

**Review by gtoison — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
+Before opening 'pull request'
```
Comment (ThrawnCA) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Needs "a":

    Before opening a 'pull request'

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
 
+** Search existing issues and pull requests to see if issue was already discussed.
```
Comment (ThrawnCA) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: "see if the issue"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
+** Check our discussions to see if issue was already discussed.
```
Comment (ThrawnCA) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: "see if the issue"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,4 +1,8 @@
...
+** Check for specific project to raise issue on that we support under [spotbugs](https://github.com/spotbugs)
```
Comment (ThrawnCA) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: "Check for a specific project that we support to raise the issue on, under"

**Review by ThrawnCA — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (hazendaz) [reply .github/PULL_REQUEST_TEMPLATE.md]: done

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (hazendaz) [reply .github/PULL_REQUEST_TEMPLATE.md]: done

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (hazendaz) [reply .github/PULL_REQUEST_TEMPLATE.md]: done

**Comment by hazendaz** [comment]
@ThrawnCA There were 4 of these, don't think we use them all - 2 were added via github recently by me so likely we never used in how github expected.  We will eventually clean that up.  For now, I did as you suggested to this PR and the 3 others I have out for same issue.  If you can, please take a look at all 4 to make sure its to your liking.  Thanks.

**Review by ThrawnCA — APPROVED** [review]

**Pull request merged**